packages/core/src/forms/control/control.element.ts

```diff
@@ -337,7 +344,7 @@ export class CdsControl extends LitElement {
   }
 
   private setupHiddenLabel() {
-    if (this.label.getAttribute('cds-layout')?.includes('display:screen-reader-only')) {
+    if (this.label?.getAttribute('cds-layout')?.includes('display:screen-reader-only')) {
       this.hiddenLabel = true;
```

Comment:
According to karma, this line here isn't covered in our unit tests. Any way we could address that?

Reply:
Yes, will take a look.